caffe2/image/image_input_op.h

```diff
@@ -515,6 +541,12 @@ bool ImageInputOp<Context>::GetImageAndLabelAndInfoFromDBValue(
   cv::Mat scaled_img;
 
   int scale_to_use = scale_ > 0 ? scale_ : minsize_;
+
+  // set the random minsize
+  if (random_scaling_) {
+    scale_to_use = std::uniform_int_distribution<>(256, 480)(*randgen);
```

Comment:
Do you think we need to give people option to tune the scaled size? Or can we assume that all the image input will be scaled between 256 to 480?